Paper.tex

```diff
@@ -904,7 +978,8 @@ \section{Contract Creation}\label{ch:create}\hypertarget{endow}{}
 F \equiv  & \big( \boldsymbol{\sigma}[a] \neq \varnothing \ \wedge\ \big(\boldsymbol{\sigma}[a]_c \neq \texttt{\small KEC}\big(()\big) \vee \boldsymbol{\sigma}[a]_n \neq 0 \big) \big) \quad \vee \\
 \nonumber &(\boldsymbol{\sigma}^{**} = \varnothing \ \wedge\ \mathbf{o} = \varnothing) \quad \vee \\
 \nonumber &g^{**} < c \quad \vee \\
-\nonumber &\lVert \mathbf{o} \rVert > 24576
+\nonumber &\lVert \mathbf{o} \rVert > 24576 \quad \vee \\
+\nonumber &\mathbf{o}[0] = \texttt{0xef}
```

Comment:
This is where EIP-3541 is incorporated (reject new contract code starting with the 0xef byte).

It is a failure condition for contract creation transactions.